ipaserver/install/cainstance.py

```diff
@@ -749,7 +789,7 @@ def __get_ca_chain(self):
 
     def __import_ca_chain(self):
         # Backup NSS trust flags of all already existing certificates
-        certdb = certs.CertDB(self.realm)
+        certdb = certs.CertDB(self.realm, self.cert_db_dir)
```

Comment:
This method is supposed to import the CA chain into the RA agent database. Now that there is no RA agent database, it should either update `/etc/ipa/ca.crt` or be removed, but it should definitely not update `/etc/httpd/alias`.